core/chains/evm/forwarders/forwarder_manager.go

```diff
@@ -113,7 +112,9 @@ func FilterName(addr common.Address) string {
 
 func (f *FwdMgr) ForwarderFor(addr common.Address) (forwarder common.Address, err error) {
 	// Gets forwarders for current chain.
-	fwdrs, err := f.ORM.FindForwardersByChain(big.Big(*f.evmClient.ConfiguredChainID()))
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Can this method be called with Forwarder closed? If not then perhaps we can use Forwarder's ctx here instead of creating a new one.